### PR TITLE
Hotfix for Story CPLP-332  - Siemens app store logo

### PR DESCRIPTION
**Pull request by fatihayyildiz**
Siemens logo implemented to their Data lake app store card

### CODE REVIEW

**Comment by fatihayyildiz** [comment]
Waiting for an approval from Siemens side 

Edit: Approved, ready for review

**Review by vsamarora — APPROVED** [review]

**Pull request merged**
